mat64/cholesky.go

```diff
@@ -14,13 +14,19 @@ import (
 	"github.com/gonum/matrix"
 )
 
-const badTriangle = "mat64: invalid triangle"
+const (
+	badTriangle = "mat64: invalid triangle"
+	badCholesky = "mat64: invalid Cholesky factorization"
+)
 
 // Cholesky is a type for creating and using the Cholesky factorization of a
 // symmetric positive definite matrix.
```

Comment:
"Cholesky methods may only be called on a value that has been successfully initialized by a call to Factorize that has returned true."